File: 15_ListVIewTodo/app/src/main/java/com/mpianatra/a15_listviewtodo/Display.java
package com.mpianatra.a15_listviewtodo;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class Display extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.display);
        String data = getIntent().getExtras().getString("name_selected", "None");
        ((TextView) findViewById(R.id.tv_display)).setText(data);
    }
}

File: Kotlin/Toast/app/src/main/java/com/example/myapplication/Display.kt
package com.example.myapplication

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class Display : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.second_activity)
    }

    fun btnClose(view: View){
        this.finish()
    }
}